README.md

```diff
@@ -62,7 +66,9 @@ stored in `conf/options.php`, which HotCRP marks as world-unreadable. You must
 ensure that your PHP can read this file.
 
 2. Edit `conf/options.php`, which is annotated to guide you.
-(`lib/createdb.sh` creates this file based on `src/distoptions.php`.)
+(`lib/createdb.sh` creates this file based on `src/distoptions.php`.) Make
+sure that `conf/options.php` is readable by the user that your webserver runs
+as.
```

Comment:
Nope, at least it didn't for me: in Ubuntu, Apache runs as `www-data`, but I installed HotCRP as `root` (in a container), and the configuration ended up being owned by `root` without read access for `www-data`.

My reading of the script is that it will set the script to be owned by the user who runs it (even if run through `sudo`); line 488 also suggests that it ought to be set `o-rwx`, but that didn't happen in my case and I had to manually make it accessible. I'll double-check, though.